pyproj/_crs.pyx

```diff
@@ -1316,20 +1339,14 @@ cdef class _CRS(Base):
 
     def to_geodetic(self):
         """
+        .. note:: This is replaced with :attr:`CRS.geodetic_crs`
 
         Returns
         -------
-        pyproj.CRS: The geographic (lat/lon) CRS from the current CRS.
+        pyproj.CRS: The geodetic CRS from this CRS.
 
         """
-        cdef PJ * projobj
-        projobj = proj_crs_get_geodetic_crs(self.projctx, self.projobj)
-        if projobj == NULL:
-            return None
-        try:
-            return self.__class__(_to_wkt(self.projctx, projobj))
-        finally:
-            proj_destroy(projobj) # deallocate temp proj
+        return self.geodetic_crs
```

Comment:
Add deprecation warning?

Reply:
Probably better to deprecated it. I had the thought as well.

Reply:
Just added deprecation warning.